dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -721,7 +721,13 @@ public void sortChildren(String inode)  throws DotDataException {
 			statement = conn.createStatement();
 			String sql;
 
-            PreparedStatement createSortPreparedStatement = conn.prepareStatement( catSQL.getCreateSortChildren() );
+            if ( DbConnectionFactory.isOracle() ){
+                //For Oracle we need to avoid ORA-01027 by creating the table before.
+                sql = catSQL.createCategoryReorderTable();
+                statement.execute( sql );

```

Comment:
Change looks good just wondering if the createCategoryReorderTable sql shouldn't be commit before the next SortChild query, if the connection is set with autocommit flag ignore my comment, thanks